sycl/test-e2e/bindless_images/vulkan_interop/sampled_images_semaphore.cpp

```diff
@@ -0,0 +1,9 @@
+// REQUIRES: cuda
```

Comment:
```suggestion
// REQUIRES: aspect-ext_oneapi_bindless_images
// REQUIRES: aspect-ext_oneapi_external_semaphore_import
```

Reply:
done. Changed to `cuda || (windows && level_zero && aspect-ext_oneapi_bindless_images)`